feature/urpf/otg_tests/urpf_vrf_test/README.md

```diff
@@ -0,0 +1,190 @@
+```
+
+## Configuration generation of DUT and ATE
+### Baseline DUT configuration
+- Configure EBGP[ASN200:ASN100] between ATE port1 – DUT port1
+- Configure IBGP[ASN100] between ATE port 2 – DUT
+- Configure a non-default vrf to host routes learned from eBGP neighborship(ATE port1) and to constraint the uRPF lookup in the non-default vrf
+- Routes in non-default VRF:
+    - Configure the static routes for `IPv4Prefix1/24` `IPv6Prefix1/64` in non-default VRF
+- DUT has DUT:Port1 and DUT:Port2 in the default network-instance
+- DUT's IP sub-interfaces belong to Default VRF
+- DUT port1 has uRPF policy at the ingress DUT:PORT1
+### Baseline ATE configuration
+- Configure EBGP[ASN200] on ATE:Port1
+#### ATE Route Advertisements:
+- ATE:Port1 advertises following valid prefixes over EBGP to DUT:Port1
+    - IPv4Prefix1/24 IPv6Prefix1/64 community 200:200
+- ATE:Port1 advertises following invalid prefixes over EBGP to DUT:Port1
+    - IPv4prefix2/24 IPv6prefix2/64
+- ATE:Port2 advertises following prefixes over IBGP to DUT:Port2
+    - IPv4prefix3/24 IPv6prefix3/64
+## Procedure:
+## URPF-1.1.1 - uRPF with valid source IP address
+    - Flow type: Native IPv4 or IPv6 traffic
+    - Simulate the below stated flows from ATE:Port1 to ATE:Port2:
+	    - IPv4Prefix1/24 to IPv4prefix3/24 at a rate of 100 packets/sec
+      - IPv6Prefix1/24 to IPv6prefix3/24 at a rate of 100 packets/sec
+    - Success Criteria:
+        - All traffic should reach port 2 and there should be no packet loss
+        - The packets sent by the sender tester is equal to the packets on the receiving tester port and also should be equal to the sum of packets seen by default.
+## URPF-1.1.2 - uRPF with invalid source IP address
+    - Flow type: Native IPv4 or IPv6 traffic
+    - The invalid prefixes should not be installed in non-default vrf
+    - Simulate the below stated flows from ATE:Port1 to ATE:Port2:
+        - IPv4prefix2/64 to IPv4prefix3/64 at a rate of 100 packets/sec
+        - IPv6prefix2/64 to IPv6prefix3/64 at a rate of 100 packets/sec
+    - Success Criteria:
+        - All traffic should be dropped by DUT since the non-default vrf couldn't validate the SIP
+        - Validation shouldn't happen against the default route present in non-default vrf
+        - The uRPF drop packet counter should increment and should be equal to the packets sent by the sender tester
+        - The packets sent by the sender tester are not equal to the packets on the receiving tester port and also the sum of packets seen by the Port2 should be zero packets.
+## URPF-1.1.3 - uRPF with valid source IP address and GUE encapsulation
+    - Flow type: Native IPv4 or IPv6 traffic encapsulated by GUE variant 1 on DUT
+    - The uRPF check should happen before the encapsulation is performed by the DUT
+    - Follow the procedure documented in [RT-3.53](https://github.com/openconfig/featureprofiles/pull/3482) for enabling GUE encapsulation on DUT
```

Comment:
Use the canonical OC from [RT-3.53](https://github.com/openconfig/featureprofiles/blob/main/feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md)